client/src/BidderBiddingList.js

```diff
@@ -1,23 +1,36 @@
 import { useEffect, useState } from "react";
-import { get } from "./TenderClient";
+import { get, post } from "./TenderClient";
 
 const BidderBiddingList = () => {
 	const [loading, setLoading] = useState(true);
 	const [bidderList, setBidderList] = useState([]);
 	const [errorMsg, setErrorMsg] = useState(null);
+	const [currentPage, setCurrentPage] = useState(1);
+	const [totalPages, setTotalPages] = useState(1);
+
+	const fetchBidderBids = async (page) => {
+		try {
+			const data = await get(`api/bidder-bid?page=${page}`);
+			setLoading(false);
+			setBidderList(data.results);
+			setTotalPages(data.pagination.totalPages);
+		} catch (error) {
+			setErrorMsg(error.message);
+		}
+	};
 
 	useEffect(() => {
-		const fetchBidderBids = async () => {
-			try {
-				const data = await get("api/bidder-bid?page=1");
-				setLoading(false);
-				setBidderList(data.results);
-			} catch (error) {
-				setErrorMsg(error.message);
-			}
-		};
-		fetchBidderBids();
-	}, []);
+		fetchBidderBids(currentPage);
+	}, [currentPage]);
+
+	const handleStatusChange = async (bidId, status) => {
+		try {
+			await post(`/api/bid/${bidId}/status`, { status });
+			fetchBidderBids(currentPage);
+		} catch (error) {
+			setErrorMsg(error.message);
```

Comment:
`error.message` doesn't exit.
```suggestion
			setErrorMsg("Server error");
```